### PR TITLE
Add tiki and fix build

### PR DESCRIPTION
**Pull request by KrisXV**

### CODE REVIEW

File: data/mods/ssb/moves.ts [hunk 1]
```diff
@@ -1205,12 +1205,10 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
...
 			onHit(target, source) {
-				switch(toID(source.types[0])) { 
-					case 'normal':
-					const typeList = ["Fighting", "Flying", "Poison", "Ground", "Rock",
-							"Bug", "Ghost", "Steel", "Fire", "Water", "Grass", "Electric",
-							"Psychic", "Ice", "Dragon", "Dark", "Fairy"];
-					const newType = typeList[this.random(typeList.length)];
+				switch (toID(source.types[0])) {
+				case 'normal':
+					const typeList = Object.keys(this.dex.data.TypeChart);
```
Comment (ghost) [inline data/mods/ssb/moves.ts:1210]: I did test this, it didn't work and caused crashes afaik

Comment (KrisXV) [reply data/mods/ssb/moves.ts:1210]: It crashed because I accidentally put Object.values and not Object.keys last time

Comment (xInstruct) [reply data/mods/ssb/moves.ts:1210]: Lemme try if this will work

Comment (ghost) [reply data/mods/ssb/moves.ts:1210]: Ohhh that makes sense

**Pull request merged**